apps/native/app/src/screens/inbox/inbox.tsx

```diff
@@ -511,41 +417,28 @@ export const InboxScreen: NavigationFunctionComponent<{
   }
 
   useEffect(() => {
-    Navigation.mergeOptions(ComponentRegistry.InboxScreen, {
-      bottomTab: {
-        iconColor: theme.color.blue400,
-        textColor: theme.shade.foreground,
-        text: intl.formatMessage({ id: 'inbox.bottomTabText' }),
-        testID: testIDs.TABBAR_TAB_INBOX,
-        iconInsets: {
-          bottom: -4,
-        },
-        icon: require('../../assets/icons/tabbar-inbox.png'),
-        selectedIcon: require('../../assets/icons/tabbar-inbox-selected.png'),
-        badge: unreadCount > 0 ? unreadCount.toString() : (null as any),
-        badgeColor: theme.color.red400,
-      },
-      topBar: {
-        rightButtons: selectState
-          ? [
-              {
-                id: ButtonRegistry.InboxBulkSelectCancelButton,
-                text: intl.formatMessage({
-                  id: 'inbox.bulkSelectCancelButton',
-                }),
-              },
-            ]
-          : [
-              {
-                id: ButtonRegistry.InboxBulkSelectButton,
-                text: intl.formatMessage({
-                  id: 'inbox.bulkSelectButton',
-                }),
-              },
-            ],
-      },
+    const topBar: OptionsTopBar = {
+      ...(isAndroid && {
+        title: { alignment: leftButtons.length > 0 ? 'center' : 'fill' },
+      }),
+      rightButtons,
+      leftButtons,
+    }
+
+    Navigation.mergeOptions(componentId, {
+      topBar,
     })
-  }, [intl, theme, unreadCount, selectState])
+  }, [componentId, rightButtons, leftButtons])
+
+  useNavigationComponentDidAppear(() => {
+    if (isAndroid) {
+      Navigation.mergeOptions(componentId, {
```

Comment:
Here is the main fix. I know it is not the most optimal solution, but it will have to do for now. Please see note in PR description why